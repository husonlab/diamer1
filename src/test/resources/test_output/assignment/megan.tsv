node id	kmer count
0	1
1	22
2	79
4	0
5	1
3	0
