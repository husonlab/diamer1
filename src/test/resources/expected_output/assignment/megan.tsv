taxid	kmer count
0	105
1	24
2	79
5	1
